packages/web3-eth/src/types.ts

```diff
@@ -77,6 +77,7 @@ export interface SendTransactionOptions<ResolveType = TransactionReceipt> {
 	transactionResolver?: (receipt: TransactionReceipt) => ResolveType;
 	contractAbi?: ContractAbi;
 	checkRevertBeforeSending?: boolean;
+	ignoreFillingGas?: boolean;
```

Comment:
```suggestion
	ignoreFillingGasLimit?: boolean;
```
is this for gas limit right?